bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/common/ThreadPoolManager.java

```diff
@@ -0,0 +1,247 @@
+                            : new int[] { Integer.valueOf(segments[0]), Integer.valueOf(segments[1]) };
+                    if (cfg[0] < cfg[1]) {
+                        logger.warn(
+                                "Ignoring invalid configuration for pool '{}': {} - max value must be bigger than min value",
+                                new Object[] { poolName, config });
+                        continue;
+                    }
+                    if (cfg[0] < 0 || cfg[1] < 0) {
+                        logger.warn("Ignoring invalid configuration for pool '{}': {} - value must not be negative",
+                                new Object[] { poolName, config });
+                        continue;
+                    }
+                    configs.put(poolName, cfg);
+                    ThreadPoolExecutor pool = (ThreadPoolExecutor) pools.get(poolName);
+                    if (pool != null) {
+                        if (pool instanceof ScheduledExecutorService) {
+                            // we only need to set the core pool size here
+                            pool.setCorePoolSize(cfg[0]);
+                            logger.debug("Updated scheduled thread pool '{}' to size {}",
+                                    new Object[] { poolName, cfg[0] });
+                        } else {
+                            // the configured pool size is the max size
+                            pool.setMaximumPoolSize(cfg[0]);
+                            pool.setCorePoolSize(cfg[1]);
+                            logger.debug("Updated thread pool '{}' to size {}-{}",
+                                    new Object[] { poolName, cfg[1], cfg[0] });
+                        }
+                    }
+                } catch (NumberFormatException e) {
+                    logger.warn("Ignoring invalid configuration for pool '{}': {} - entries must be integer values",
+                            new Object[] { poolName, config });
+                    continue;
+                }
+            }
+        }
+    }
+
+    /**
+     * Returns an instance of a scheduled thread pool service. If it is the first request for the given pool name, the
+     * instance is newly created.
+     *
+     * @param poolName a short name used to identify the pool, e.g. "discovery"
+     * @return an instance to use
+     */
+    static public ScheduledExecutorService getScheduledPool(String poolName) {
+        ExecutorService pool = pools.get(poolName);
+        if (pool == null) {
+            synchronized (pools) {
+                // do a double check if it is still null or if another thread might have created it meanwhile
+                pool = pools.get(poolName);
+                if (pool == null) {
+                    int[] cfg = getConfig(poolName);
+                    pool = Executors.newScheduledThreadPool(cfg[0], new NamedThreadFactory(poolName));
```

Comment:
No, if no configuration is there, it will go create it with the default values.
But yes, the pools are configurable through Configuration Admin service as a standard mechanism.